components/Card/card.css

```diff
@@ -0,0 +1,229 @@
+    .card p{
+        margin: 0;
+        padding:1rem ;
+        color: var(--gray-color);
+    }
+
+    .card img{  
+        width: 100%;
+        border-bottom: 2px solid var(--primary-color);
+    }
+
+    .cancelled{
+        text-decoration: line-through;
+    }
+
+    .discount{
+        color: var(--error-color);
+    }
+    .gray{
+       color: var(--gray-color);
+    }
+
+
+/* .card-product{
+    height: max-content;
+     width: max-content;
+     box-shadow: 1px 1px 1px var(--gray-color);
+     margin: 1rem;
+}   
+    .card-product img {
+        display: flex;
+        height: 12rem;
+        width: 100%;
+        border-bottom: 3px solid var(--primary-color);
+    }
+    .card-product h1,.card-product p,h2{
+        font-size: 1rem;
+        padding: 0 1rem 0.2rem;
+    }
+    .card-product button{
+        margin: 0 1rem 0.5rem;
+        padding: 0.4rem;
+        cursor: pointer;
+        border: none;
+        background-color: var(--primary-color);
+        color: var(--white-color);
+        border-radius: 5px;
+    } */
+/* card product */
+    .card-product{
+        height: max-content;
+        width: 20rem;
+        box-shadow: 1px 1px 1px var(--gray-color);
+        margin: 1rem;
+        position: relative;
+   }   
+       .card-product img {
+           display: flex;
+           height: 12rem;
+           width: 100%;
+           border-bottom: 3px solid var(--primary-color);
+       }
+       .card-product h1,.card-product p,h2{
+           font-size: 1rem;
+           padding: 0 1rem 0.2rem;
+       }
+        .card-product button{
+           margin: 0 1rem 0.5rem;
+           padding: 0.4rem;
+           border-radius: 5px;
+   
+       } 
+       .middle-section{
+           height: max-content;
+           width: 100%;
+           align-self: center;
+       }
+       .last-section{
+           padding: 1rem 0 0 0;
+       }
+       .price-tag{
+           width: max-content;
+           background: var(--primary-color);
+           position: relative;
+           bottom: 35px;
+           color: var(--white-color);
+       }
+       .discount-tag{
+           width: max-content;
+           background: var(--primary-color);
+           position: relative;
+           bottom: 183px;
+           left: 267px;
+           color: var(--white-color);
+       }
+      
+       .card-sideImage{
+           display: flex;
+           flex-direction: row;
+           align-items: center;
+           width: max-content;
+           height: max-content;
+           
+       }
+           .card-sideImage img{
+               width: 60%;
+               border: none;
+           }
+
+/* card horizontal */
+.card-horizontal{
+    height: 10rem;
+    width: 30rem;
+    display: flex;
+    flex-direction: column;
+    box-shadow: 1px 1px 1px var(--gray-color);
+    margin: 1rem;
+}    
+    .card-header{
+        display: flex;
+        width: 100%;
+    }
+        .card-header img{
+            width: 100%;
+        }
```

Comment:
suggestion: maintain indentation of code
not gonna affect anything but will increase code readability